### PR TITLE
Unsupported examples

### PR DESCRIPTION
**Pull request by tqrahman**
**What is is PR for?**

- [ ] Code update
- [X] Documentation update
- [X] Infrastructure update

**What does this PR do?**  
Removed examples that the CDP wouldn't support. Removed 3d print files for enclosures.

**Is this related to an open issue?**  
https://github.com/ClusterDuck-Protocol/ClusterDuck-Protocol/issues/376
https://github.com/ClusterDuck-Protocol/ClusterDuck-Protocol/issues/378
https://github.com/ClusterDuck-Protocol/ClusterDuck-Protocol/issues/417

**Testing methodology**  
Nothing to test. No code was changed. Examples were removed and readmes' were updated.

**Additional context**  
Add any other technical detail or considerations here.

**Checklist**
Before you submit this pull request, please make sure you have done the following:

_General_  

- [X] Contribution Guidelines: Have you read the contribution guidelines?
- [X] Code of Conduct: Have you read the code of conduct?
- [X] Documentation: If applicable, have you added or updated all relevant documentation?

_For Code Updates_  

- [ ] Hardware Validation: If relevant, have you validated your changes on actual supported Duck hardware?
- [ ] Unit Tests: Have you run the unit tests on the device?
- [ ] Network Testing: If applicable, have you tested your changes on a Duck network?
- [ ] Licensing: Have you added a copyright and license header to each new file?

_Tested Targets (Please check all that apply)_

- [ ] All
- [ ] Heltec LoRa v3
- [ ] Heltec LoRa v2
- [ ] Heltec CubeCell Series
- [ ] TTGO T-Beam (SX1262)
- [ ] Others

### CODE REVIEW

**Comment by amirna2** [comment]
@tqrahman The  pipeline is attempting to build the `1.Ducks/Ducklink` and `1.Ducks/MamaDuck`
And it will build the two examples for both the ESP32 and the Heltec CubeCell

The ESP32 build is failing because of a compile error in CDP.
You can see the build logs here: https://github.com/ClusterDuck-Protocol/ClusterDuck-Protocol/actions/runs/9558992094/artifacts/1611239813

```
5222CBD8EC665BE670FEF82A92C0027F/file_opts /home/runner/Arduino/libraries/ClusterDuck_Protocol/src/Ducks/Duck.cpp -o /dev/null
Alternatives for esp_int_wdt.h: []
ResolveLibrary(esp_int_wdt.h)
  -> candidates: []
/home/runner/Arduino/libraries/ClusterDuck_Protocol/src/Ducks/Duck.cpp:5:12: fatal error: esp_int_wdt.h: No such file or directory
    5 |   #include <esp_int_wdt.h>
      |            ^~~~~~~~~~~~~~~
compilation terminated.
```

This post here may explain why the build is failing and how to potentially fix it.
https://forum.arduino.cc/t/esp32-cam-cant-compile-a-project-from-github/1220008

**Comment by amirna2** [comment]
To fix this you can try the suggested change here:
https://docs.espressif.com/projects/esp-idf/en/v5.0/esp32/migration-guides/release-5.x/system.html#esp-system

TLDR;

Change the include from `#include <esp_task_wdt.h>` to `#include "esp_private/esp_int_wdt.h"`
The reason being that in ESP32 core, the header file is not longer accessible, you can still include it, but you have to specify the location.

**Comment by faradaym** [comment]
@amirna2 
> but there is an underlying issue with CDP. What is `#include <esp_int_wdt.h>` used for in CDP and is it required in the first place?

I have searched the history to see where this was added, and I found it [here](https://github.com/ClusterDuck-Protocol/ClusterDuck-Protocol/commit/7e0c0280e651f373305758aa88e26704e22c6605#diff-b4120167dac9b929ffcba424ef79f98fa5925a842976627048d03de4d13728cc). It looks like it is for Over the Air firmware updates, so I think we can safely remove the header and related code in Duck.cpp

**Comment by faradaym** [comment]
@amirna2 there are still issues with the v3 build. I'm not able to see any details beyond that it failed on DuckLink

**Comment by hawk4031** [comment]
After fixing errors in the build, we were able to get the builds to pass finally.
